tests/async/tasyncfile.nim

```diff
@@ -41,11 +41,11 @@ proc main() {.async.} =
     await file.write("test2")
     file.close()
     file = openAsync(fn, fmWrite)
-    await file.write("test3")
+    await file.write("t3")
     file.close()
     file = openAsync(fn, fmRead)
     let data = await file.readAll()
-    doAssert data == "test3"
+    doAssert data == "t3"
```

Comment:
The original test worked only because `test3` is of the same length as `test2`, so it was overwritten. But if you write `t3` without truncating, the result would be `t3st2`.